### DISCUSSION TITLE
The ElastAlert does not reach the threshold and sends an alarm.

### DISCUSSION THREAD

**Discussion by Cloud-Ding** [jertel/elastalert2#138] [Q&A] [answered]
my template.yaml

timestamp_type: custom
timestamp_format: '%Y-%m-%dT%H:%M:%S%z'
type: frequency
num_events: 500
timeframe:
    hours: 1

filter:
- query:
    query_string:
      query: "offering_name:Kepler AND log_content:\"FAILED\""

When I receive an alarm, I query the alarm through kibana.
 
Search criteria ： offering_name:Kepler AND log_content:"FAILED"
hit 87 hits in an hour's range

So I want to know what caused this?

**Comment by ferozsalam** [comment, marked as answer]
My first suggestion would be to make sure you are doing a Lucene syntax query in Kibana, as opposed to a KQL query, and to check that the number of hits returned is still lower than expected with the Lucene query.

There should be a small box on the right hand side of the search bar showing either 'KQL' or 'Lucene', click on it to change it if it's showing KQL.

**Reply by ferozsalam** [reply]
Another potential point is the formatting of the `query` string - you probably want to escape the double quotes (if you really need them) like so:

`query: "offering_name:Kepler AND log_content:\"FAILED\""`

If there's no particular reason for using the double quotes, doing the below might also work:

`query: "offering_name:Kepler AND log_content:FAILED"`

**Reply by Cloud-Ding** [reply]
Thank you for your advice, but I get the same hits with Lucene query and KQL;

In addition, the double quotes have no impact on the query.
This problem occurs occasionally. I reproduce the problem locally and generate alarms. Therefore, the problem is blocked.

**Comment by tomfotherby** [comment]
Did you ever figure out your problem? Because I have the same thing (https://github.com/jertel/elastalert2/discussions/1163)
